Lib/pdb.py

```diff
@@ -491,6 +491,8 @@ def interaction(self, frame, tb_or_exc):
                 Pdb._previous_sigint_handler = None
 
         _chained_exceptions, tb = self._get_tb_and_exceptions(tb_or_exc)
+        if isinstance(tb_or_exc, BaseException) and tb is None:
+            raise ValueError("No exception traceback to inspect")
```

Comment:
This should never happen when called from `post_mortem()` as `post_mortem()` ensure that at least the main exception has a tb, let me know what you wish to do here.

One possibilty is to automatically jump to the any exception with a traceback in `_chained_exceptions`

Reply:
Not sure we should put this piece of code in there. This should in theory NEVER happen if the user does not directly use `interaction()`. A heuristic does not help much if the case should not happen. The corner case stil has a corner case - what if there's no exception that has traceback? You'll still error out in the same manner. So we put effort into a super corner case but still do not fix it for good.

I suggest we simply put an assertion there to make sure the main exception has a traceback - that's a requirement for all the future users and developers as this is a private function. It would make the code cleaner and we have one less thing to worry about.

Reply:
That is fair. Relevant code and test removed, and added the assertion.